src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -750,8 +750,8 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
       return { icon, label: isReviewSheetOpen ? quoteError : reviewLabel, disabled: true, type: 'hold' };
     }
 
-    if (hasEnoughFundsForGas.value === false) {
-      const nativeCurrency = RainbowNetworkByChainId[sellAsset?.chainId || ChainId.mainnet].nativeCurrency;
+    if (!hasEnoughFundsForGas.value === false) {
```

Comment:
oh you want to remove the `!` here that was introduced

Reply:
💀